lochness/transfer/__init__.py

```diff
@@ -375,6 +375,8 @@ def create_s3_transfer_table(Lochness, rewrite=False) -> None:
                     continue
                 try:
                     source = re.search(r'upload: (\S+)', line).group(1)
+                    # if not source.startswith('/'):
+                        # source = '/' + source
```

Comment:
I would remove this (unintentional) dead code.

Reply:
done